docs/modules/reference/pages/performance-data-collection/collectors/SnmpCollector.adoc

```diff
@@ -89,6 +89,30 @@ Ignored when using Newts as time-series storage.
 .Configuration overview for SNMP data collection
 image::performance-management/collectors/01_snmp-datacollection-configuration.png[]
 
+=== Support for opaque data types
+
+OpenNMS supports opaque data types including MIB import and data collection.
+Opauqe is ASN.1(Abstract Syntax Notation One) binary encoded data which is additionally wrapped in a hex string.
+The extracted from out of a hex string data can be of various types. 
```

Comment:
```suggestion
OpenNMS interprets these encoded values and properly compiles them so that users do not need to edit the resulting compiled data collection definitions.
```